cogs/yugioh.py

```diff
@@ -0,0 +1,225 @@
+import json
+import discord
+import aiohttp
+import Paginator
+from discord.ext import commands
+from discord import app_commands
+ 
+class Yugioh(commands.Cog, description="Yugioh commands."):
+    
+    def __init__(self, client: commands.Bot):
+        self.client = client
+        
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print("Yugioh module has been loaded.")
+
+    @commands.command(aliases=["scn"], brief="Searches a card by name.", description="A command that helps you search a Yugioh card.")
+    async def searchcard(self, ctx: commands.Context, *card: str):
```

Comment:
What does the asterisk before the card argument do?

Reply:
The asterisk is use as a filler to take the rest of the argument and create a tuple of all those entries. It is used here if the user desires to enter a yugioh card that has spaces in its name. Using the following command will give you...

Ex: 

Inp => #scn Blackwing - Zephyros The Elite
Out => card = ("Blackwing", "-", "Zephyros", "The", "Elite")

instead of only 
Out => card = "Blackwing"

if you were to only use the card:str without the asterisk